INFO     core.jenkins_utils:jenkins_utils.py:210 running cleanup
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 403 None
DEBUG    core.jenkins_utils:jenkins_utils.py:92 failed to get page on the first try, retrying after login
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /login?from=%2F HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /j_spring_security_check HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /job/MyProject/doDelete HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/securityRealm/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /computer/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/credentials/store/system/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /me/my-views/view/all/submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /user/admin/appearance/configSubmit HTTP/1.1" 200 132
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /user/admin/security/ HTTP/1.1" 200 None
INFO     core.jenkins_utils:jenkins_utils.py:201 deleting token with uuid fa408a50-6016-428a-bd24-f5a44ffbe2e0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /user/admin/descriptorByName/jenkins.security.ApiTokenProperty/revoke HTTP/1.1" 200 0
DEBUG    conftest:conftest.py:90 Argument --remote-allow-origins=* added to the chrome
DEBUG    conftest:conftest.py:90 Argument --disable-gpu added to the chrome
DEBUG    conftest:conftest.py:90 Argument --no-sandbox added to the chrome
DEBUG    conftest:conftest.py:90 Argument --disable-dev-shm-usage added to the chrome
DEBUG    conftest:conftest.py:90 Argument --headless=new added to the chrome
DEBUG    conftest:conftest.py:90 Argument --window-size=1920,1080 added to the chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:101 Selenium Manager binary found at: /opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:114 Executing process: /opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager --browser chrome --debug --language-binding python --output json
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Found chromedriver 140.0.7339.207 in PATH: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 chrome detected at /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Running command: /usr/bin/google-chrome --version
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Output: "Google Chrome 140.0.7339.207 "
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Detected browser: chrome 140.0.7339.207
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Required driver: chromedriver 140.0.7339.207
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Driver path: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Browser path: /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.service:service.py:231 Started executable: `/usr/bin/chromedriver` in a child process with pid: 5714 using 0 to output -3
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:50827
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50827 "POST /session HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /crumbIssuer/api/json HTTP/1.1" 200 155
INFO     tests.api.support.base_api:base_api.py:52 Generating Jenkins API token at: "/generateNewToken"
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /user/admin/descriptorByName/jenkins.security.ApiTokenProperty/generateNewToken HTTP/1.1" 200 167
INFO     tests.api.support.base_api:base_api.py:63 Generated token: 1108...
INFO     tests.api.tests_ui.organization_folder.conftest:conftest.py:23 Creating Organization Folder 'Organization Folder Project Name' via API POST request.
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /createItem?name=Organization%20Folder%20Project%20Name HTTP/1.1" 200 0
INFO     tests.api.tests_ui.organization_folder.conftest:conftest.py:39 Organization Folder 'Organization Folder Project Name' was created successfully.
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50827 "POST /session/3cdfa56daaa7161b118179c707558b27/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50827 "GET /session/3cdfa56daaa7161b118179c707558b27/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50827 "POST /session/3cdfa56daaa7161b118179c707558b27/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50827 "POST /session/3cdfa56daaa7161b118179c707558b27/element/f.170CCBE1BB1850ED14B9126B89C90FB7.d.C7B8F10F12DC3A3279574EC5014F3491.e.1/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50827 "POST /session/3cdfa56daaa7161b118179c707558b27/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50827 "POST /session/3cdfa56daaa7161b118179c707558b27/element/f.170CCBE1BB1850ED14B9126B89C90FB7.d.C7B8F10F12DC3A3279574EC5014F3491.e.3/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50827 "POST /session/3cdfa56daaa7161b118179c707558b27/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50827 "POST /session/3cdfa56daaa7161b118179c707558b27/element/f.170CCBE1BB1850ED14B9126B89C90FB7.d.C7B8F10F12DC3A3279574EC5014F3491.e.8/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50827 "GET /session/3cdfa56daaa7161b118179c707558b27/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50827 "GET /session/3cdfa56daaa7161b118179c707558b27/cookie HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /crumbIssuer/api/json HTTP/1.1" 200 155
DEBUG    core.jenkins_utils:jenkins_utils.py:31 update_crumb: 7d88548a2f1f5aa07ed41f033456298e80f5454fdd4d133168bd00aa13044b2b
DEBUG    LoginPage:login_page.py:32 login crumb: 7d88548a2f1f5aa07ed41f033456298e80f5454fdd4d133168bd00aa13044b2b
DEBUG    MainPage:ui_element.py:28 waiting for element_to_be_clickable(('xpath', "//table[@id='projectstatus']//tbody//td[3]/a"),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50827 "POST /session/3cdfa56daaa7161b118179c707558b27/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50827 "POST /session/3cdfa56daaa7161b118179c707558b27/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50827 "GET /session/3cdfa56daaa7161b118179c707558b27/element/f.170CCBE1BB1850ED14B9126B89C90FB7.d.1CE1520C60E0867946669AD5DD65DB11.e.64/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50827 "POST /session/3cdfa56daaa7161b118179c707558b27/element/f.170CCBE1BB1850ED14B9126B89C90FB7.d.1CE1520C60E0867946669AD5DD65DB11.e.64/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50827 "GET /session/3cdfa56daaa7161b118179c707558b27/url HTTP/1.1" 200 0
DEBUG    OrganizationFolderPage:ui_element.py:28 waiting for visibility_of_element_located(('tag name', 'h1'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50827 "POST /session/3cdfa56daaa7161b118179c707558b27/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50827 "POST /session/3cdfa56daaa7161b118179c707558b27/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50827 "GET /session/3cdfa56daaa7161b118179c707558b27/element/f.170CCBE1BB1850ED14B9126B89C90FB7.d.7F3D14A38E8A7545191513E72345DABC.e.95/text HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:50827 "DELETE /session/3cdfa56daaa7161b118179c707558b27 HTTP/1.1" 200 0